modules/test/playwright/tests/object-web/objectLayout.spec.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {editObjectDefinitionPagesTest} from '../../fixtures/editObjectDefinitionPagesTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {objectPagesTest} from '../../fixtures/objectPagesTest';
+import getRandomString from '../../utils/getRandomString';
+import getRandomObjectFieldText from './utils/getRandomObjectFieldText';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	editObjectDefinitionPagesTest,
+	featureFlagsTest({
+		'LPS-187854': true,
+	}),
+	loginTest(),
+	objectPagesTest
+);
+
+test.describe('Manage Object Layouts through the Object Layout tab', () => {
+	test('Can view all fields of an object when creating its layout', async ({
```

Comment:
Hey @vngoliveira, to follow the pattern with other tests, could you please start the describe and test name using lowercase letters? Thanks 😄